### PR TITLE
replace TASK_UNIQUE_INDEX with INDEX

### PR DESCRIPTION
**Pull request by potatowagon**

### CODE REVIEW

**Comment by potatowagon** [comment]
add labels

File: docs/UserGuide.md [hunk 1]
```diff
@@ -417,11 +417,11 @@ TaskTracker.
 
 Delete a useless floating task/event/deadline on TaskTracker.
 
-    del-float|-deadline|-event <TASK_UNIQUE_INDEX>
+    del-float|-deadline|-event <INDEX>
 
 * Every event, deadline and floating task will have their own unique index.
 
-* A task's `TASK_UNIQUE_INDEX` will be the same as what is the number beside the task name.
+* A task's `INDEX` is the number beside the task name.
```
Comment (pyokagan) [inline docs/UserGuide.md:424]: I'll just tweak this to become

```
A task's `INDEX` is the number displayed beside the task name in the user interface
```

**Review by pyokagan — APPROVED** [review]
Thanks.

**Comment by pyokagan** [comment]
Merged in  6d9efd431e9d95766d100af751c9ce15065a269a

**Pull request closed without merging**